typescript/infra/scripts/get-owner-ica.ts

```diff
@@ -0,0 +1,82 @@
+import { ethers } from 'ethers';
+
+import { AccountConfig, InterchainAccount } from '@hyperlane-xyz/sdk';
+import { assert, eqAddress, rootLogger } from '@hyperlane-xyz/utils';
+
+import { getSecretRpcEndpoints } from '../src/agents/index.js';
+
+import { getArgs as getEnvArgs, withChainRequired } from './agent-utils.js';
+import { getEnvironmentConfig, getHyperlaneCore } from './core-utils.js';
+
+function getArgs() {
+  return withChainRequired(getEnvArgs())
+    .option('ownerChain', {
+      type: 'string',
+      description: 'Origin chain where the governing owner lives',
+      demandOption: true,
+    })
+    .option('owner', {
+      type: 'string',
+      description:
+        'Address of the governing owner. Defaults to the owner of the owner chain',
```

Comment:
not sure I fully follow the description, what's "owner of the owner chain"

Reply:
was trying to say the owner in the owner.ts file relating to the ownerChain passed in, but this is confusing, changing this